### ISSUE TITLE
Append Noctua/GO-CAM models into main GAF/GPAD output files

### ISSUE THREAD

**Issue by kltm** [geneontology/pipeline#66]
Just as how we add PAINT files to the main GAFs during processing, we'd like to do the same with the Noctua annotations that we generate.

Closing this would be taking the outputted models from #65 and performing a similar check and appending process, including adding additional header information to the final outputted GAF/GPAD.

Depends on #65

**Comment by dougli1sqrd** [comment]
All changes in go-site will happen in a branch to be created
Changes in ontobio should be non-breaking and continually merged into master when features are finished.
[ ] Set up Noctua GPAD in the config yamls as a `group` called `gocam`. URLs here should not be absolute, to reflect that they will be created during a pipeline run. Whatever specific mechanism that this uses this (ontobio + go-site) will have to change along side this.
[ ] Teach ontobio to read the updated config format so ontobio can process just-created noctua gpad
[ ] Ontobio should have a basic (simple) Association model that both parsers and writers can use instead of coincidental dictionaries. 
[ ] Validate.py in Ontobio needs to be able to arbitrarily mix in any number of other group sources in processing a top level group resource. This represents generically mixing in sources like paint, but without hard coding them. As in the future we'll have more than just noctua and paint.

**Comment by kltm** [comment]
Could you name the branch in go-site `issue-pipeline-66-go-cam-append`?

**Comment by dougli1sqrd** [comment]
I locally already created a branch so very close to your suggestion. I can make it your name.

**Comment by kltm** [comment]
I'm just trying to encourage a pattern so we can visually match branches easily: I've defaulted to `issue-[remote-repo-name-]<issue-number>-<a few human readable words>`

**Comment by dougli1sqrd** [comment]
gocam.yaml in the go-site branch: https://github.com/geneontology/go-site/blob/issue-pipeline-66-go-cam-append/metadata/datasets/gocam.yaml

Ontobio just had merged into master ability to read relative URLs in metadata group yamls.

**Comment by kltm** [comment]
I believe we're done here.
